addons/crm/static/src/scss/crm.scss

```diff
@@ -31,3 +31,10 @@
         }
     }
 }
+
+.o_crm_merge_post_message {
+    .o_crm_merge_content {
```

Comment:
Can you use `pl-3` and `border-left border-dark` instead ?

Reply:
Indeed, actually the "blockquote" element adds its own padding, so I removed it.
This was a leftover from NEB's implementation.